migrations/000001_bookstable.up.sql

```diff
@@ -1,7 +1,26 @@
+/*
+CREATE OR REPLACE FUNCTION trigger_set_timestamp()
+RETURNS TRIGGER AS $$
+BEGIN
+  NEW.updated_at = NOW();
+  RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+*/
+
 CREATE TABLE IF NOT EXISTS public.bookstable
 (
     id uuid PRIMARY KEY NOT NULL,
     name text   UNIQUE NOT NULL,
     price numeric(6,2),
-    inventory integer
-)
\ No newline at end of file
+    inventory integer,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
Should we let Postgresql to deal with the date/time data or our api should create this data and just store it on db?

Reply:
Either is fine. There is good and bad thing in both.
We just need to be consistent, if you are doing it in the Go code always do it that way, if the queries are taking care of it, always do it that way.
I like having it done in the Go side, it makes you not depend as much in the database functionality.

Reply:
I think that doing it in API will be easier, too. There is some preferable function to write the timestamp or should I discover it on my own?

Reply:
Use `go` `time.Time` type. SQL driver should be smart enough to know how to convert that.
Just make sure you are always working in the UTC timezone.